src/commands/createNewProject/ProgrammingModelStep.ts

```diff
@@ -29,7 +29,9 @@ export class ProgrammingModelStep extends AzureWizardPromptStep<IProjectWizardCo
             }
         });
 
+        const learnMoreQp = { label: localize('learnMore', '$(link-external) Learn more about Model V4...'), description: '', data: undefined };
```

Comment:
Does the `IAzureQuickPick` lack a way to override the "Learn More" text? That would be a good addition to the API that could simplify this code.

Reply:
I think at the moment we only have a Learn More `MessageItem` for notification dialogs and a Learn More `QuickInputButton` which is that little ? button on the top right of the quickpick window: 
![image](https://user-images.githubusercontent.com/5290572/222807982-f25aef43-c089-427e-ab08-b1c5fc660e8f.png)

Today, we don't have a standardized way of adding `learn more...` to quick picks, but I think it makes sense to add that to the API along with a way to override the learn more text.